components/Choice/index.scss

```diff
@@ -128,3 +128,142 @@
     }
   }
 }
+
+[data-options-per-row] {
+  flex-wrap: wrap;
+
+  .radioButton {
+    label {
+      border-top: 0;
+    }
+
+    // Reset default border-radius for first and last child
+    &:first-child {
+
+      label {
+        border-radius: 0;
+      }
+    }
+
+    &:last-child {
+
+      label {
+        border-radius: 0;
+      }
+    }
+  }
+
+  // Mobile Styling
+  @media (max-width: $screen-sm-max) {
+    .radioButton {
+      flex: 0 0 50%;
+
+      // First Row
+      &:nth-child(-n+2) {
+        label {
+          border-top: 1px solid $blue;
+        }
+      }
+
+      // All Left Radio Buttons
+      &:nth-child(2n + 1) {
+        label {
+          border-left: 1px solid $blue;
+        }
+      }
+
+      // Style for border corners
+      // Top Left Corner
+      &:first-child {
+        label {
+          border-top-left-radius: $border-radius-base;
+        }
+      }
+
+      // Top Right Corner
+      &:nth-child(2) {
+        label {
+          border-top-right-radius: $border-radius-base;
+        }
+      }
+
+      // Bottom Left Corner
+      &:nth-child(2n + 1):nth-last-child(-n + 2) {
+        label {
+          border-bottom-left-radius: $border-radius-base;
+        }
+      }
+
+      // Bottom Right Corner
+      &:last-child,
+      &:nth-child(2n):nth-last-child(-n + 2) {
+        label {
+          border-bottom-right-radius: $border-radius-base;
+        }
+      }
+    }
+  }
+}
+
+@mixin options-per-row($count) {
+  [data-options-per-row='#{$count}'] {
+    @media (min-width: $screen-sm-max) {
```

Comment:
You will want to use $screen-md-min with min-width or else there will be a 1px error at 997px.

*With bootstrap variables you always need to use -min with min width and -max with max widths